README.md

```diff
@@ -1,17 +1,42 @@
 # poao-endringslogg-sanity
 
-Sanity-oppsett for PO arbeidsoppfølgings endringslogg.
-Innholder er tilgjengelig på [https://endringslogg.sanity.studio/production/structure](https://endringslogg.sanity.studio/production/structure)
+Sanity-oppsett for PO arbeidsoppfølging sin endringslogg.
+Innholdet er tilgjengeleg på [https://endringslogg.sanity.studio/production/structure](https://endringslogg.sanity.studio/production/structure)
 
 ## Bygg og deploy
+For å få nye endringar i main ut i prod må appen deployast manuelt frå eiga maskin.
 
-Appen deployes fra lokal maskin ved å kjøre `(npx) sanity deploy`
+Før du kan gjere dette må du gje brukaren din rette tilgangar og logge inn i Sanity frå terminalen din.
+
+### Tilgangar
+Gå til sida for endringsloggen (lenka frå fyrste avsnitt), logg inn med SSO for navikt. Dette gjev lesetilgang til brukaren din.
+
+Utviklarar treng rollene "Administrator" og "Developer".
+For å redigere innhald trengs "Editor".
+
+Be nokon i teamet som har Administrator-rolla om å oppdatere rollane til brukaren din.
+
+Du kan sjekke kven som har kva tilgangar på organisasjons-sidene til Sanity: https://www.sanity.io/organizations/ojSsHMQGf/project/li581mqu/members-v2
```

Comment:
Legge ved ein kommentar om at ein her må velje "Sign in with SSO" og skrive inn `navikt` som "Organization slug".